docs/source/data/data-sources.mdx

```diff
@@ -247,6 +247,7 @@ class MoviesAPI extends RESTDataSource {
   }
 }
 ```
+> Note the use of (`encodeURIComponent`)[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent]. This is a standard JavaScript function that encodes special characters in a URI, preventing an attack vector known as "Web Parameter Tampering". For more information on this vulnerability, see [this article](https://owasp.org/www-community/attacks/Web_Parameter_Tampering).
```

Comment:
I'm not sure that the linked owasp article explains the same thing — that seems to be about things like "folks assume that you can put a hidden input in a page and then trust its value" rather than about *encoding* of parameters...

Reply:
Swapped article reference to a small example of URL injection